index.html

```diff
@@ -94,24 +102,23 @@ <h2 class="heading-underline">Upcoming events</h2>
 					<div class="articles row">
```

Comment:
if you use a `row` you **must** have `col-...` elements inside. 

Just like you do here: https://github.com/CodeYourFuture/bikes-for-refugees/pull/33/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR74